tests/system/test_limit_eps/api_configurations/allowed_limit_eps.yaml

```diff
@@ -0,0 +1,5 @@
+wazuh-manager:
+  upload_configuration:
+    limits:
+      eps:
+        allow: yes
```

Comment:
Instead of creating one configuration file for each value, we could put a placeholder like `ALLOW` and then modify it in the tests cases, like it's done [here](https://github.com/wazuh/wazuh/blob/master/tests/integration/test_api/test_config/test_cache/data/configuration_templates/configuration_cache.yaml).